<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDeployerDialog</class>
 <widget class="QDialog" name="AddDeployerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Deployer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="2">
      <widget class="ValidatingLineEdit" name="source_path_field"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="source_dir_label">
       <property name="text">
        <string>Source directory:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="ValidatingLineEdit" name="path_field"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="toolTip">
        <string>Name of the deployer</string>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="toolTip">
        <string>This is where installed mods will be deployed to</string>
       </property>
       <property name="text">
        <string>Target directory:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="toolTip">
        <string>Determines how mods will be deployed</string>
       </property>
       <property name="text">
        <string>Deployer type:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="file_picker_button">
       <property name="toolTip">
        <string>Pick target directory</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="folder-open"/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="type_box"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="method_label">
       <property name="toolTip">
        <string>Determines how files will be deployed.</string>
       </property>
       <property name="text">
        <string>Deployment method:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="source_picker_button">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="folder-open"/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="ValidatingLineEdit" name="name_field"/>
     </item>
     <item row="4" column="2">
      <widget class="QComboBox" name="deploy_mode_box">
       <item>
        <property name="text">
         <string>Hard Link</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sym Link</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Copy</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="unsafe_sorting_box">
     <property name="toolTip">
      <string>When enabled: Sorting plugins uses LOOT. This will affect override order. 
When disabled: Sorting only groups plugins by types. Does not affect override order.</string>
     </property>
     <property name="text">
      <string>Use LOOT for sorting</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="rev_depl_ignore_cb">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Adds all files currently in the target directory to a ignore list.&lt;br&gt;Does not include files managed by other deployers.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Ignore current files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="rev_depl_separate_cb">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use separate directories for managed files for every profile.&lt;br&gt;This can be used to manage multiple versions of the same files.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Separate directories for profiles</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="rev_depl_ignore_button">
     <property name="toolTip">
      <string>Update the list of ignored files with all files currently in the target directory. Does not include files managed by other deployers.</string>
     </property>
     <property name="text">
      <string>Update Ignored Files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="warning_label">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:700; color:#ff0000;&quot;&gt;Warning: Enabling copy deployment will double the disk space required per mod and may drastically increase deployment time!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextInteractionFlag::LinksAccessibleByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="sym_link_label">
     <property name="text">
      <string>Sym link deployment works across filesystems, unlike hard link deployment. If this causes issues, try hard links instead.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>14</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header>ui/validatinglineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddDeployerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddDeployerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
